### PR TITLE
uthenticode: init at 1.0.4

### PR DESCRIPTION
**Pull request by arcz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/uthenticode/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    owner = "trailofbits";
+    repo = "uthenticode";
+    rev = "v${version}";
+    sha256 = "16j91cki63zk4d7wzwvq8al98l8hmvcdil3vfp44ink4q4bfswkx";
+  };
+
+  patches = [
+    # adds USE_SYSTEM_GTEST cmake flag, the patch won't be necessary in next versions
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/trailofbits/uthenticode/pull/40.patch";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/uthenticode/default.nix:17]: Please fetch the commit.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109378` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>uthenticode: log was empty</li>
  </ul>
</details>

hash mismatch

```
hash mismatch in fixed-output derivation '/nix/store/m31blrhgagqknbzc0yavhpv3kmz2vmfd-40.patch':
wanted: sha256:1gnxv286z9mwkzr5jxsic5f8qxg83rpsf08npin5h6m9kznjfwaj
got:    sha256:0smafyfml72kzzrzqgjgrhbjkixm683a7ljv3vmisl2m6iikll05
cannot build derivation '/nix/store/9f9yahnbwm39mz7i2rc6iw3sn4f2xrgn-uthenticode-1.0.4.drv': 1 dependencies couldn't be built 
cannot build derivation '/nix/store/g4ha7g1vr4g8wb5bz0gqgfqddpc3hraa-env.drv': 1 dependencies couldn't be built
```

File: pkgs/development/libraries/uthenticode/default.nix (hunk 1 above)
Comment (arcz) [reply pkgs/development/libraries/uthenticode/default.nix]: done

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109378` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>uthenticode</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109378` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>uthenticode</li>
</ul>
</details>

**Pull request merged**